addons/payment/views/payment_form_templates.xml

```diff
@@ -408,37 +407,66 @@
-            <t t-call="payment.availability_report_button"/>
+            <strong>No payment method available</strong>
             <br/>
-            If you believe that it is an error, please contact the website administrator.
-            <div class="mt-2" groups="base.group_system">
-                <a
-                    t-if="request.env.company.country_id.is_stripe_supported_country"
-                    name="activate_stripe"
-                    href="/odoo/action-payment.action_activate_stripe"
-                    role="button"
-                    class="btn btn-primary"
-                > ACTIVATE STRIPE </a>
-                <a
-                    role="button"
-                    type="action"
-                    class="btn-link alert-warning ps-2"
-                    href="/odoo/action-payment.action_payment_provider"
-                >
-                    <strong><i class="oi oi-arrow-right"></i> View alternatives </strong>
-                </a>
-            </div>
+            <t t-if="request.env.is_system()">
+                <p t-if="providers_sudo">
+                    None is configured for
+                    <t t-out="request.env.company.country_id.name"/>,
+                    <t t-out="request.env.company.currency_id.name"/>.
+                </p>
+                <p t-else="">
+                    No payment providers are configured.
+                </p>
+                <div class="mt-2" groups="base.group_system">
+                    <a
+                        t-if="request.env.company.country_id.is_stripe_supported_country
+                            and not providers_sudo"
+                        name="activate_stripe"
+                        href="/odoo/action-payment.action_activate_stripe"
+                        role="button"
+                        class="btn btn-primary"
+                    > ACTIVATE STRIPE </a>
+                    <a
+                        t-if="availability_report"
+                        role="button"
+                        class="btn-link alert-warning"
+                        data-bs-toggle="collapse"
+                        href="#payment_availability_report"
+                    >
+                        <strong><i class="fa fa-file-text"/> Show availability report</strong>
+                    </a>
+                    <a
+                        t-if="not providers_sudo"
+                        role="button"
+                        type="action"
+                        class="btn-link alert-warning pe-2"
+                        href="/odoo/action-payment.action_payment_provider"
+                    >
+                        <strong><i class="fa fa-eye"/> Payment Providers</strong>
+                    </a>
+                    <a
+                        t-else=""
+                        role="button"
+                        type="action"
+                        class="btn-link alert-warning ps-2"
```

Comment:
As suggested, only use `pe-2`, it will be much easier to get it right than check for every combination that the padding is as expected with `ps-2` classes.